sos/report/plugins/apt.py

```diff
@@ -47,15 +47,27 @@ def setup(self):
 
     def postproc(self):
         super().postproc()
+
+        common_regex = r"(http(s)?://)\S+:\S+(@.*)"
+        common_replace = r"\1******:******\3"
+
         self.do_file_sub(
             "/etc/apt/sources.list",
-            r"(deb\shttp(s)?://)\S+:\S+(@.*)",
-            r"\1******:******\3"
+            common_regex, common_replace
         )
         self.do_path_regex_sub(
             "/etc/apt/sources.list.d/",
-            r"(deb\shttp(s)?://)\S+:\S+(@.*)",
-            r"\1******:******\3"
+            common_regex, common_replace
+        )
+
+        self.do_file_sub(
+            "/etc/apt/apt.conf",
+            common_regex, common_replace
+        )
+
+        self.do_path_regex_sub(
+            "/etc/apt/apt.conf.d/",
+            common_regex, common_replace
```

Comment:
If you apply same obfuscation four times, maybe you can shorten it to

    for p in ["/etc/apt/sources.list", ..]:
        self.do_file_sub(p, common_regex, common_replace)

?